project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("org.scalajs.junit.JUnitFramework"),
```

Comment:
Could we use the same package name as the standard sbt testing interface for JUnit on the JVM? So that we don't have to write this.